src/components/ChatEntry.js

```diff
@@ -1,22 +1,46 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
+import { useState } from 'react';
+
+
+const ChatEntry = ({sender, body, id, timeStamp, liked, updateChatData}) => {
+  const user ='Vladimir';
+  const [like, setLike] = useState(liked);
```

Comment:
You've used a ternary operator to figure out the sender in the line below. You could do the exact same for the likes as well!